sql/_13_issues/_24_1h/cases/cbrd_25290.sql

```diff
@@ -0,0 +1,39 @@
+-- This test case verifies the following issue: CBRD-25290.
+-- After giving grants to a user, if that user is deleted, the corresponding grantee in the _db_auth tables remain null.
+-- Ensure the core is no longer dumped and the expected query result is returned
+
+CREATE USER user_select;
+CREATE USER user_insert;
+CREATE USER user_update;
+CREATE USER user_delete;
+CREATE USER user_alter;
+CREATE USER user_index;
+CREATE USER user_execute;
+CREATE USER user_all_privileges;
+
+DROP TABLE IF EXISTS tbl;
+CREATE TABLE tbl (a INT);
+
+GRANT SELECT ON tbl to user_select;
+GRANT SELECT ON tbl to user_insert;
+GRANT SELECT ON tbl to user_update;
+GRANT SELECT ON tbl to user_delete;
+GRANT SELECT ON tbl to user_alter;
+GRANT SELECT ON tbl to user_index;
+GRANT SELECT ON tbl to user_execute;
+GRANT SELECT ON tbl to user_all_privileges;
+
```

Comment:
GRANT SELECT ON tbl to user_select;
GRANT INSERT ON tbl to user_insert;
GRANT UPDATE ON tbl to user_update;
GRANT DELETE ON tbl to user_delete;
GRANT ALTER  ON tbl to user_alter;
GRANT INDEX  ON tbl to user_index;
GRANT EXECUTE ON tbl to user_execute;
GRANT ALL PRIVILEGES ON tbl to user_all_privileges;

Reply:
The wrong version had been commited; I have revised the file to the correct version in the latest commit.

Reply:
An incorrect version of the file had been committed; I have revised the file to the correct version in the latest commit.